packages/rich-text/src/plugins/EmbeddedEntityBlock/Util.ts

```diff
@@ -21,19 +22,22 @@ export async function selectEntityAndInsert(
     baseConfig.entityType === 'Asset' ? dialogs.selectSingleAsset : dialogs.selectSingleEntry;
   const config = { ...baseConfig, withCreate: true };
 
-  const { selection } = editor;
+  const rteSlide = watchCurrentSlide(sdk.navigator);
```

Comment:
What happens in Compose?

Reply:
In this case, yes. There are no slide events.

In some cases, e.g. calling `sdk.navigator.openEntry(entryId, {slideIn: true})` we open  the web app instead, `{slideIn: true}` is simply ignored and also doesn't trigger any slide-in events.